### PR TITLE
Tasklist feature

### PR DESCRIPTION
**Pull request by guerreiropedr0**
In this pull request, I added the functionality of the task list:
- You can add tasks;
- You can edit tasks;
- You can remove single tasks;
- You can check completed tasks, and clear them all.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+.task-list {
+  display: flex;
+  flex-direction: column;
+}
+
+.tasks {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (VSeray) [inline src/index.css:31]: It would be better if you add more styling to your project. It would be good practice for using CSS.

**Review by VSeray — CHANGES_REQUESTED** [review]
### **STATUS:** CHANGES REQUIRED ♻️ ♻️

Hi @guerreiropedr0,

Good job so far!
✔️ Good documentation
✔️ Props and states are used
However, there are some issues that you still need to work on to go to the next project but you are almost there!

###  **Required** Changes ♻️

Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

image
- This is what I received when I opened the app it would be better to fix every linter even on the app try `npx eslint . --fix`.

image
- As you can see in the image above whenever I input a task it is not being saved to the local storage. It would be best if you could add a Local storage to your project.

File: src/index.css (hunk 1 above)
Comment (guerreiropedr0) [reply src/index.css:31]: Thank you for the review, I only implemented the functionality in this PR.

Comment (VSeray) [reply src/index.css:31]: Okay

**Review by VSeray — APPROVED** [review]
### Hi @guerreiropedr0 
👋,

**Congratulations! 🎉 Project Approved** ✅

No linter errors ✔️
Descriptive commit messages ✔️
Well written PR ✔️
Proper use of git-flow ✔️

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**